src/js/panels.js

```diff
@@ -119,10 +119,176 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         that.locate("meter").css("height", percentage + "%");
     };
 
+    fluid.defaults("gpii.firstDiscovery.panel.keyboard", {
+        gradeNames: ["fluid.prefs.panel", "autoInit"],
+        preferenceMap: {
+            "gpii.firstDiscovery.stickyKeys": {
+                "model.stickyKeys": "default"
+            }
+        },
+        selectors: {
+            input: ".gpiic-fd-keyboard-input",
+            instructions: ".gpiic-fd-keyboard-instructions",
+            assistance: ".gpiic-fd-keyboard-assistance"
+        },
+        selectorsToIgnore: ["assistance"],
+        events: {
+            onOfferAssistance: null,
+            onInitInput: null
+        },
+        model: {
+            // offerAssistance: boolean
+            // tryAccomodation: boolean
+            input: ""
+        },
+        components: {
+            assistance: {
+                type: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster",
+                createOnEvent: "onOfferAssistance",
+                container: "{that}.container",
+                options: {
+                    messageBase: "{keyboard}.options.messageBase",
+                    model: {
+                        tryAccomodation: "{keyboard}.model.tryAccomodation"
+                    },
+                    modelRelay: [{
+                        source: "stickyKeysEnabled",
+                        target: "{keyboard}.model.stickyKeys",
+                        forward: "liveOnly",
+                        singleTransform: {
+                            type: "fluid.transforms.identity"
+                        }
+                    }, {
+                        source: "stickyKeysEnabled",
+                        target: "{keyboardInput}.model.stickyKeysEnabled",
+                        backward: "liveOnly",
```

Comment:
However, since neither stickyKeysAdjuster nor keyboardInput seem to specify an initial value for stickyKeys, it would seem that both of these "liveOnly" qualifications could be removed.